src/irr/asset/CImageWriterJPG.cpp

```diff
@@ -105,6 +105,7 @@ static bool writeJPEGFile(io::IWriteFile* file, const asset::CImageData* image,
 	switch( image->getColorFormat () )
 	{
         case asset::EF_R8G8B8_UNORM:
+		case asset::EF_R8G8B8_SRGB:
 			format = video::CColorConverter::convert_R8G8B8toR8G8B8;
```

Comment:
We want to get rid of CColorConverter, it is legacy code from old irrlicht and doesn't have good format coverage/support.

Maybe you can use the `decode` and `encode` functions that @Crisspl  has made.

JPEG only supports SRGB without conversion (natively), if you encounter `EF_R8G8B8_UNORM` or anything similar then you need a convert to `EF_R8G8B8_SRB` from the input format.

However you have my approval to error-out in such case.